lips/lip-7.md

```diff
@@ -0,0 +1,53 @@
+---
+lip: 7
+title: Technical Working Group Lifecycle
+authors: Viktor Bunin of Bison Trails (@viktorbunin)
+status: Draft
+type: Process
+created: 10/30/2020
+---
+
+
+## Abstract
+This LIP outlines the process by which technical working groups (TWG’s) are formed and how they should conduct their operations.
+
+## Formation of a Technical Working Group
+Technical Working Groups (TWG) are formed to address or explore specific technical topics or issues as they relate to Libra. TWGs collect information and expert input on a particular topic, discuss material changes to the Libra Payment Network (LPN), and create outputs in the form of recommendations, reports, and Libra Improvement Proposals (LIPs).
+
+The proposal to form a technical working group can be made by any Association member. The proposal must include, but is not limited to:
```

Comment:
Do we want to close this off to only association members? I could see an argument for and against this.  Just curious what the reasoning was that you went with this model

Reply:
This is a great point. I was thinking originally that it would just be association members, but I don't see a truly stellar reason to restrict it. If for some reason there is a problem with having it be open, it can always be restricted in the future. Updating this!